chia/wallet/did_wallet/did_wallet.py

```diff
@@ -441,8 +441,8 @@ def create_backup(self) -> str:
         """
         assert self.did_info.current_inner is not None
         assert self.did_info.origin_coin is not None
-        output_str = f"{self.did_info.origin_coin.parent_coin_info}:"
-        output_str += f"{self.did_info.origin_coin.puzzle_hash}:"
+        output_str = f"{self.did_info.origin_coin.parent_coin_info.hex()}:"
+        output_str += f"{self.did_info.origin_coin.puzzle_hash.hex()}:"
```

Comment:
Why is this required here?

Reply:
our fixed-size byte types print as hex, whereas the built-in prints as binary.

Reply:
I should have read the initial description it seems :) Hmm.. so we have `byte` objects but it's hinted as `bytes32`. Shouldn't we also just hint `bytes` instead like with `int`/`uint64` since its confusing when you look at the definition of `Coin` and it appears to be `bytes32` what it isn't?

Reply:
the reason I hinted the puzzle hash and parent id as `bytes32` despite actually being `bytes` (but guaranteed to be 32 bytes) was that there would have been a *lot* of places requiring a cast in this patch. In almost all of those cases the cast wouldn't technically be necessary, because our `bytes32` type is so similar to `bytes`